### DISCUSSION TITLE
ImportError

### DISCUSSION THREAD

**Discussion by freedomforsnails** [AbdBarho/stable-diffusion-webui-docker#718] [General]
Hi,

`docker compose --profile auto up --build` gives this error: ` ImportError: cannot import name 'TypeIs' from 'typing_extensions' (/opt/conda/lib/python3.10/site-packages/typing_extensions.py)
auto-1 exited with code 1`

If anyone can point me towards a solution I'd be very grateful.

Thanks

**Comment by mikekline** [comment]
getting the same error as well

**Comment by Wayne-O07** [comment]
Same error here too

**Comment by vshis** [comment]
The solution by yasu-nxt in https://github.com/AbdBarho/stable-diffusion-webui-docker/issues/719 worked for me
